mods/default/depends.txt

```diff
@@ -0,0 +1 @@
+players?
```

Comment:
Is this needed at all now? Seems to me default should always be the first mod loaded?